### ISSUE TITLE
Input validation sniff overlooking usage of vars not within a function parenthesis

### ISSUE THREAD

**Issue by shadyvb** [WordPress-Coding-Standards/WordPress-Coding-Standards#117]
This causes notices and fails validation

``` php
$foo = $_REQUEST['bar'];
```

Targeted sniff is ValidatedSanitizedInput
